restapi/header/header.avsc

```diff
@@ -4,8 +4,13 @@
   "name": "Header",
   "doc": "Metadata for Dataset. It is a summary explaining what kind of information are stored in the Dataset.",
   "fields": [
-    {"name": "descriptiveStatistic", "type": "DescriptiveStatistic" },
-    {"name": "unit", "type": "org.radarcns.avro.restapi.sensor.Unit"},
-    {"name": "effectiveTimeFrame", "type": "EffectiveTimeFrame" }
+    {"name": "subjectId", "type": "string", "doc": "Subject identifier"},
+    {"name": "sourceId", "type": "string", "doc": "Source identifier"},
+    {"name": "source", "type": "org.radarcns.avro.restapi.source.SourceType", "doc": "Source type"},
+    {"name": "sensor", "type": "org.radarcns.avro.restapi.sensor.SensorType", "doc": "Sensor that has generated data"},
```

Comment:
Could this field and the next four also be `null`? That could perhaps also be the default? Is this header only for statistics, or also possibly for raw data (like phone status)?